src/server/models/thinky.js

```diff
@@ -1,61 +1,13 @@
 import dumbThinky from 'rethink-knex-adapter'
 import redis from 'redis'
 import bluebird from 'bluebird'
+import config from '../knex.js'
 
 bluebird.promisifyAll(redis.RedisClient.prototype)
 bluebird.promisifyAll(redis.Multi.prototype)
 
-// // This was how to connect to rethinkdb:
```

Comment:
Has all of this been lifted verbatim into `knex.js` as a refactor? Or is there a bigger difference that I'm missing?

Reply:
Should be a verbatim copy